app/helpers/carts_helper.rb

```diff
@@ -0,0 +1,9 @@
+module CartsHelper
+  def cart_summary(items)
+    cart_summary = 0
+    items.each do |item|
+      cart_summary += (item.itemable.real_price * item.quantity.to_i)
```

Comment:
create instance method inside `order_detail_item model`

def total_price
   itemable.real_price * quantity
end

In cart_summary method

```
def cart_summary(items)
    cart_summary = 0
    items.each do |item|
      cart_summary += item.total_price
    end
    return cart_summary
  end
end
```